src/maintainer/conda_forge_yml.rst

```diff
@@ -353,6 +354,50 @@ defaults are as follows:
       # conda-forge-ci-setup-feedstock references
       tooling_branch_name: main
 
+.. _github_actions:
+
+github_actions
```

Comment:
Should we add a note about how MS-hosted GHA workflows are not currently supported for feedstocks, beyond a few specific core ones?

Reply:
Done.